src/components/GameCard/GameCard.tsx

```diff
@@ -10,7 +10,7 @@ import ImageWithLoader from "../ui/Image/ImageWithLoader";
 import { ProgressBar } from "../ui/ProgressBar/ProgressBar";
 import { WishButton } from "../ui/WishButton/WishButton";
 
-import { PromoBloc } from "./PromoBloc/PromoBloc";
+import PriceBox from "./PriceBox/PriceBox";
```

Comment:
Tout les components importer doivent être entre crochet
```suggestion
import { PriceBox } from "./PriceBox/PriceBox";
```